app/components/post_list/post/body/reactions/reaction.tsx

```diff
@@ -51,6 +51,15 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
 
 const Reaction = ({count, emojiName, highlight, onPress, onLongPress, theme}: ReactionProps) => {
     const styles = getStyleSheet(theme);
+    const digits = String(count).length;
+    const containerStyle = useMemo(() => {
+        const minWidth = 50 + (digits * 5);
```

Comment:
Magic number ` * 5` ...does it stand for the max digits for the number of reactors?